content/en/blog/2023/security-audit-23.md

```diff
@@ -0,0 +1,158 @@
+---
+title: Security Audit '23
+description: Explore our latest security audits and findings.
+type: docs
```

Comment:
```suggestion
```

Reply:
When you specify `type:docs` you're preventing the Docsy blog layout from being used.